subgraphs/_reference_/src/sdk/protocols/perpfutures/poolSnapshot.ts

```diff
@@ -41,9 +42,9 @@ export class PoolSnapshot {
     if (!this.pool._lastUpdateTimestamp) return;
```

Comment:
This is probably worthy of a `log.error` since it would mean something else is pretty wrong